gpustat/core.py

```diff
@@ -221,6 +249,14 @@ def _repr(v, none_value='??'):
             reps += "%(FSpeed)s{entry[fan.speed]:>3} %%%(C0)s, "
 
         reps += "%(CUtil)s{entry[utilization.gpu]:>3} %%%(C0)s"
+        if "enc" in show_codec:
```

Comment:
I think it's good to wrap them with parenthesis when `--show-codec` is specified, like the first version.

Reply:
Ok, I added the parenthesis back.